darts/models/torch_forecasting_model.py

```diff
@@ -1215,6 +1221,9 @@ def _get_batch_prediction(self, n: int, input_batch: Tuple, roll_size: int) -> T
 
 
 class DualCovariatesTorchModel(TorchForecastingModel, ABC):
+
+    uses_past_covariates = False
```

Comment:
Same here?

Reply:
from one of your earlier comments:
> Here's the list:
* `PastCovariatesTorchModel`: only past
* `FutureCovariatesTorchModel`, `DualCovariatesTorchModel` and `DualCovariatesForecastingModel`: only future
* `MixedCovariatesTorchModel` and `SplitCovariatesTorchModel`: both

Reply:
All good.